README.md

```diff
@@ -132,3 +132,43 @@ This sketch simplification code is  freely available for free non-commercial
 use, and may be redistributed under these conditions. Please, see the [license](/LICENSE)
 for further details.
 
+## Deploy with Docker
+
+### Dependencies
+- [Docker](https://www.docker.com/)
+
+### Build Local Image
+
+The models will automatically be downloaded into the container, which may take a few minutes.
+
+```
+git clone https://github.com/bitstrider/sketch_simplification.git
+cd sketch_simplification
+docker build -t bitstrider/sketch_simplification .
+```
+
+### Run with Interactive Shell
+
+```
+docker run -i -v bitstrider/sketch_simplification
+```
+
+Once inside the interactive shell, start using the library as usual.
+
+```
+./figs.sh
+python simplify.py --img test_line.png --out out_rough.png --model model_pencil2.t7
+```
+
```

Comment:
After building this image and running it interactively, inside the shell, running figs.sh gives:

root@da71b5a52837:/workspace/sketch_simplification# ./figs.sh
Processing fig01_eisaku.png...Traceback (most recent call last):
  File "simplify.py", line 4, in <module>
    from torch.utils.serialization import load_lua
ModuleNotFoundError: No module named 'torch.utils.serialization'
root@da71b5a52837:/workspace/sketch_simplification#